airflow/providers/databricks/hooks/databricks.py

```diff
@@ -388,6 +437,32 @@ def repair_run(self, json: dict) -> None:
         """
         self._do_api_call(REPAIR_RUN_ENDPOINT, json)
 
+    def get_cluster_state(self, cluster_id: str) -> ClusterState:
+        """
+        Retrieves run state of the cluster.
+
+        :param cluster_id: id of the cluster
+        :return: state of the cluster
+        """
+        json = {"cluster_id": cluster_id}
+        response = self._do_api_call(GET_CLUSTER_ENDPOINT, json)
+        state = response["state"]
+        state_message = response["state_message"]
+        return ClusterState(state, state_message)
+
+    async def a_get_cluster_state(self, cluster_id: str) -> ClusterState:
```

Comment:
wdyt about this name?
```suggestion
    async def async_get_cluster_state(self, cluster_id: str) -> ClusterState:
```

I know that we have `_a_do_api_call`, but since this will be a public method, we can improve its name and make it clearer.

Reply:
OK, I will change the names more clearer, thanks! 👍